### PR TITLE
Added Inf home to some providers and new providers

### LINKED ISSUE #1354: YOMOVIES and HDMOVIE source not showing episodes
### Source information

hdmovie and yomovies

### Source link

http://hdmovie2.org    and   https://yomovies.vip

### Steps to reproduce

**source-** hdmovie and yomovies

**Problem-** both showing web series as movies only playing the last episodes I am aware that many series(old series) are bundled together like movies but most series and all new series are in episodes, it only plays last uploaded episode and these are only Hindi sources so I have no other option please fix it. 🤞

### screenshots
**-----------------------------------------------------------------------------------------**
### what it shows⬇
Screenshot_20220728-072536

### what it plays⬇
Screenshot_20220728-073838

### what actually is⬇
Web capture_28-7-2022_73444_yomovies vip

### **more screenshots**
Screenshot_20220728-073605

Screenshot_20220728-073626

Web capture_28-7-2022_73216_yomovies vip

### CloudStream version

3.0.0 and pre-release build

### Android version

Andriod 10 and 8

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to pre-release version **[Latest](https://github.com/LagradOst/CloudStream-3/releases)**.
- [X] If related to a provider, I have checked the site and it works, but not the app.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by hexated**
- [WCO Fun](https://www.wcofun.com)
- [Duboku](https://www.duboku.tv) - **Chinese**
- [Kisskh](https://kisskh.me)
- added extractor
- fixes #1354

### CODE REVIEW

**Comment by LagradOst** [comment]
Ait, this is a lot. Will review tomorrow 👍

File: app/src/main/java/com/lagradost/cloudstream3/movieproviders/KisskhProvider.kt [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+        return newTvSeriesLoadResponse(
+            res.title ?: return null,
+            url,
+            if (res.type == "Movie" || episodes.size == 1) TvType.Movie else TvType.TvSeries,
+            episodes
+        ) {
+            this.posterUrl = res.thumbnail
+            this.year = res.releaseDate?.split("-")?.first()?.toIntOrNull()
+            this.plot = res.description
+            this.tags = listOf("${res.country}", "${res.status}", "${res.type}")
```
Comment (LagradOst) [inline app/src/main/java/com/lagradost/cloudstream3/movieproviders/KisskhProvider.kt:94]: why this????

**Review by LagradOst — APPROVED** [review]
LGTM. really nice done 👍

**Comment by hexated** [comment]
now it's ready

**Pull request merged**